### ISSUE TITLE
Tutorial on Variational Inference

### ISSUE THREAD

**Issue by nishantb21** [tensorflow/probability#149]
The current tutorials cover a majority of MCMC. Could we get one for variational inference? The edward tutorial on Supervised Learning shows how to run inference using Kullback-Leibler divergence. It would be great if you could provide a similar port over at TFP.

**Comment by adam2392** [comment]
Do you still need help on this? I would like to contribute here.

**Comment by nishantb21** [comment]
Yes! I would love to have a tutorial for variational inferencing. Thanks!

**Comment by mr-easy** [comment]
Is there any simple VI tutorial available now?

**Comment by willtai** [comment]
Is this issue still open? I would like to contribute some simple tutorials 😄

**Comment by mr-easy** [comment]
@willtai Yes, I guess it's still open. And people still need some tutorials :)

**Comment by Tasfia-Ara** [comment]
@nishantb21 I would like to contribute to this as well!
